crates/uv/src/commands/project/init.rs

```diff
@@ -47,7 +47,17 @@ pub(crate) async fn init(
     };
 
     // Make sure a project does not already exist in the given directory.
-    if path.join("pyproject.toml").exists() {
+    if project_kind == InitProjectKind::Script {
+        if explicit_path.is_none() {
+            anyhow::bail!("Missing script name to initialize",);
+        }
+        if path.exists() {
```

Comment:
Should we use `try_exists` here?

Should we add metadata to the front of the script if it doesn't exist? Or should we tackle that in a separate pull request? I think we already have machinery for this for `uv add`